packages/eslint-plugin/project.json

```diff
@@ -68,6 +68,30 @@
           "npx ts-node --project ./tsconfig.tools.json ./tools/scripts/generate-rule-docs.ts eslint-plugin"
         ]
       }
+    },
+    "check-rule-lists": {
+      "dependsOn": [
+        {
+          "target": "update-rule-lists"
+        }
+      ],
+      "executor": "nx:run-commands",
+      "options": {
+        "parallel": false,
+        "commands": [
+          "git diff --quiet --exit-code packages/eslint-plugin/README.md || (echo \"\n-------\n\nCHECK RULE LISTS ERROR: You have uncommitted changes to the generated rule lists.\n\nYou should run `yarn update-rule-lists` and commit the result.\n\n-------\n\" && exit 1)"
+        ]
+      }
+    },
+    "update-rule-lists": {
+      "executor": "nx:run-commands",
+      "outputs": ["{projectRoot}/docs"],
```

Comment:
```suggestion
      "outputs": ["{projectRoot}/README.md"],
```

Reply:
This script does not produce outputs in the docs dir, it only updates the README.md in the project root right?